src/Modules/Installer/templates/step1.html.twig

```diff
@@ -24,14 +24,14 @@
     <div class="fork-module-content">
       <input type="hidden" name="step" value="2" />
       <div id="requirementsInformation">
-        {% for requirementCategory in checker.requirementCategories %}
+        {% for requirement_category in checker.requirementCategories %}
```

Comment:
Shouldn't `checker.requirementCategories` be changed into `checker.requirement_categories`

Reply:
I don't think in this case since it is calling a method and I don't think the underscored name will work